docs/virtualization_troubleshooting.md

```diff
@@ -0,0 +1,55 @@
+# Troubleshooting virtualization backup and restore
+
+1. [Create a backup for a single VM in a namespace with multiple VMs.]( #create-a-backup-for-a-single-vm-in-a-namespace-with-multiple-vms)
+1. [Create a restore for a single VM in a backup with multiple VMs.](#create-a-restore-for-a-single-vm-in-a-backup-with-multiple-vms)
+
+## Create a backup for a single VM in a namespace with multiple VMs.
+
+If you have a namespace with many VMs, and only want to back up one of them, you can use a label selector with the `app: vmname` label to indicate which VM should be included in the backup. For example:
+```
```

Comment:
lets add syntax highlighting for yaml in the 3 code blocks?

Reply:
Sure, done.